### ISSUE TITLE
Android Add CV (Doc)/Pic(jpg/png)/grademark report (PDF)

### ISSUE THREAD

**Issue by NajibAbuSamra** [shalomweiss/mm-system-2017#45]
*discussion here*

close when finished

**Comment by masterFoad** [comment]
No gif support, only jpg png for Images.
It can be added though. @NajibAbuSamra

**Comment by NajibAbuSamra** [comment]
@masterFoad  no need, I think jpg is good enough, it's the common case for img type, no?

**Comment by masterFoad** [comment]
@NajibAbuSamra  Indeed it is.

**Comment by NajibAbuSamra** [comment]
@masterFoad  @Minitour  what's the status on this task? a bit hard to follow on android progress since I don't see the changes in github 😜

**Comment by NajibAbuSamra** [comment]
@masterFoad @Minitour  can we close this?

**Comment by masterFoad** [comment]
@NajibAbuSamra you can close.
